ADAL/src/request/ADAuthenticationRequest+Broker.m

```diff
@@ -306,14 +307,25 @@ - (void)callBroker:(ADAuthenticationCallback)completionBlock
     id<ADTokenCacheDataSource> dataSource = [_tokenCache dataSource];
     if (dataSource && [dataSource isKindOfClass:[ADKeychainTokenCache class]])
     {
+        NSString* keychainGroup = [(ADKeychainTokenCache*)dataSource sharedGroup];
+        NSString* teamId = [ADKeychainUtil keychainTeamId:&error];
```

Comment:
Okay sounds good. I forgot the method was called `retrieveTeamIDFromKeychain` so it doesn't matter if the rest of the app group name isn't the right "adalcache" one.